thank-you.php

```diff
@@ -0,0 +1,22 @@
+<?php
+    require_once __DIR__.'/_backend/preload.php';
+    $page['title'] = 'Thank You for downloading';
+    $page['theme-color'] = '#3E4E54';   
+    
+    include $template['header'];
+    include $template['alert'];
+?>
+
+<section class="hero">
+    <div class="grid">
+    <div>
+        <img src="<?=$sitewide['root'];?>images/brand/logomark.png" alt="logomark">
+        <h1>Thank you for downloading!</sup></h1>
+        <h4>For help and more info please see the installation guide</h4>
```

Comment:
Grammatical suggestion:
> For help and more info, please see the installation guide

Note the added comma. I'm no grammar genius, but it *feels* more natural to me.